src/main/java/hudson/remoting/MimicException.java

```diff
@@ -11,6 +17,7 @@
  * @deprecated Use {@link ProxyException} instead.
  */
 @Deprecated
+@Restricted(DoNotUse.class)
```

Comment:
Yes. `@Restricted` will just make a whatever external dependency to fail. But yes, it does not make much difference